osmUtils/utils_osm.py

```diff
@@ -345,17 +348,18 @@ def download_OSM(
     
     try:
         response_json = []
-        for polygon_coord_str in geometry_coord_str:
-            print(polygon_coord_str)
-            query_str = f'{overpass_settings};({filters}(poly:"{polygon_coord_str}");>;);out;'
-            print(query_str)
-            print(f'Requesting data within polygon from API in {len(polygon_coord_str)} request(s)')
-            response_j = overpass_request(
-                        query_str, 
-                        timeout=timeout, 
-                        overpass_endpoint=overpass_endpoint
-                    )
-            response_json.append(response_j)
+        for filter_ in filters:
```

Comment:
for a temp object use `_<name>`, in this case `_filter`

Reply:
ok, it's changes now! :)